doc/releases/release-notes-3.5.rst

```diff
@@ -388,6 +422,12 @@ Networking
 * Wi-Fi
   * Added Passive scan support.
   * The Wi-Fi scan API updated with Wi-Fi scan parameter to allow scan mode selection.
+  * Updated TWT handling.
+  * Added support for generic network manager API.
+  * Added support for W-Fi mode setting and selection.
```

Comment:
apologies if I missed this one earlier. I will approve nevertheless as we can also address pending typos one last time before the release, I guess.

```suggestion
  * Added support for Wi-Fi mode setting and selection.
```